lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -21,6 +21,13 @@ all_interfaces:
   get_context: ~
   get_value: '/^interface (.*)/'
 
+bfd_echo:
+  _exclude: [ios_xr]
+  kind: boolean
+  get_value: '/^(?:no )?bfd echo$/'
+  set_value: '<state> bfd echo'
+  default_value: true
```

Comment:
This one is tricky, I think. Is the default state always true, or just when feature bfd is enabled?